docs/api/version-history.md

```diff
@@ -21,6 +21,7 @@ keywords: "API, Docker, rcli, REST, documentation"
   and `OperatingSystem` if the daemon was unable to obtain this information.
 * `GET /info` now returns information about the product license, if a license
   has been applied to the daemon.
+* `GET /nodes` now supports a filter type `node-label`  filter to filter nodes based on the node-label. The format of the label filter could be `label=<key>`/`label=<key>=<value>` to remove those with the specified labels, or `label!=<key>`/`label!=<key>=<value>` to remove those without the specified labels.
```

Comment:
The examples here still don't match; e.g. `label=<key>=<value>` should be either `node-label=<key>=<value>` or `node.label=<key>=<value>`